WarOfEmpires.QueryHandlers/Players/GetPlayersQueryHandler.cs

```diff
@@ -24,9 +24,8 @@ public sealed class GetPlayersQueryHandler : IQueryHandler<GetPlayersQuery, List
                     Id = p.Id,
                     DisplayName = p.DisplayName,
                     Population = p.Farmers + p.WoodWorkers + p.StoneMasons + p.OreMiners + p.Peasants + p.SiegeEngineers
-                        + p.Archers.Soldiers + p.Archers.Mercenaries
-                        + p.Cavalry.Soldiers + p.Cavalry.Mercenaries
-                        + p.Footmen.Soldiers + p.Footmen.Mercenaries
+                        + p.Troops.Sum(t => t.Soldiers)
+                        + p.Troops.Sum(t => t.Mercenaries)
```

Comment:
The Players page fails to load, and throws an exception here (can't cast null to Int32) due the Soldiers and Mercenaries being null for all players who have not yet trained troops. Is that just on my end to initiate values for all players in the troops table, or does the null exception need to be handled here?

Reply:
That was a weird one, it did this apparently because the sum gets executed in SQL. Fixed, but the fix is not pretty.